tests/ui/lifetimes/issue-105227.stderr

```diff
@@ -9,13 +9,13 @@ LL |
 LL |     v.0.chars().chain(v.1.chars())
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
-help: to declare that `impl Iterator<Item = char>` captures `'_`, you can introduce a named lifetime parameter `'a`
+help: add a `use<...>` bound to explicitly capture `'_`
    |
-LL | fn chars0<'a>(v :(&'a  str, &'a str)) -> impl Iterator<Item = char> + 'a  {
```

Comment:
We however have lost the suggestion to give the lifetimes a name so they can be referenced in the opaque, so this `+ use<'_>` is now wrong. I could recover it, but would like to do it asynchronously :)